packages/core/lib/commands/help/displayCommandHelp.js

```diff
@@ -6,7 +6,19 @@ module.exports = async function (selectedCommand, subCommand, options) {
 
   const chosenCommand = commands[selectedCommand].meta;
 
-  if (subCommand && chosenCommand.subCommands[subCommand]) {
+  if (subCommand) {
+    if (!chosenCommand.subCommands) {
+      console.log(`WARNING: ${selectedCommand} does not have sub commands.`);
+      process.exit();
```

Comment:
How do you feel about echoing what the user entered to let them know it is an invalid command? For example, if the input is `truffle help networks disconnected`:

WARNING: **truffle networks disconnected** is not a valid command  vs
WARNING: **networks does not have sub commands**

Reply:
This might be a good spot to throw a `@truffle/error`. @eggplantzzz ?

Reply:
I like the idea of echoing what the user entered. Would the messaging be too general? If we use the not a valid command message, I can merge the two cases together(when the `chosenCommand` doesn't have subcommand and when `subCommand` is not valid. 
What would the benefit be to throw a `@truffle/error`?